Makefile

```diff
@@ -39,7 +39,7 @@ test:
 	docker-compose run --rm gem bash -c "bundle exec rake test"
 
 lint:
-	docker-compose run --rm gem bash -c "bundle exec rubocop"
+	docker-compose run --rm gem bash -c "bundle exec rubocop -A"
```

Comment:
Is this used in CI/CD pipeline? Maybe we need a separate command named `lint_fix`?

Reply:
No